zerver/lib/push_notifications.py

```diff
@@ -449,9 +450,20 @@ def get_apns_payload(message: Message) -> Dict[str, Any]:
         'custom': {
             'zulip': {
                 'message_ids': [message.id],
+                'sender_email': message.sender.email,
+                'sender_id': message.sender.id
```

Comment:
Like Tim said above, let's design this API to work well with multiple accounts, because that will be important soon. That calls for identifying two things: the server, and the realm.

One way to do this would be to just send the value of `message.sender.realm.host` -- this should be the same value that the user had to enter to connect to the server when they first set up the app, like `chat.zulip.org` or `example.zulipchat.com`.

Another would be to identify the server and the realm separately -- this could look like
```
'server': settings.EXTERNAL_HOST,
'realm_id': message.sender.realm.id,
```
As I think about it, that might be most robust -- it'll avoid disruption when e.g. a realm at `example.zulipchat.com` moves to its own domain like `chat.example.org`, just like we avoid disruption in the web app in that case. Let's do that, unless we see a reason to do something else.

Reply:
Ok, will address this!